common/djangoapps/entitlements/models.py

```diff
@@ -302,6 +320,17 @@ def get_active_entitlements_for_user(cls, user):
             enrollment_course_run=None
         ).select_related('user').select_related('enrollment_course_run')
 
+    @classmethod
+    def get_most_recent_entitlement(cls, user, course_uuid):
```

Comment:
Is this getting used anywhere?

Reply:
Nope. At some point it was, but I don't remember where / why. I'll cut it.